pyEpiabm/pyEpiabm/intervention/case_isolation.py

```diff
@@ -0,0 +1,51 @@
+#
+# Case isolation Class
+#
+
+import random
+
+from pyEpiabm.intervention import AbstractIntervention
+
+
+class CaseIsolation(AbstractIntervention):
+    """Case isolation intervention
+    """
+
+    def __init__(
+        self,
+        isolation_duration,
+        isolation_delay,
+        isolation_probability,
+        isolation_effectiveness,
+        isolation_house_effectiveness,
+        population,
+        *args,
+        **kwargs
+    ):
+        self.isolation_duration = isolation_duration
+        self.isolation_delay = isolation_delay
+        self.isolation_probability = isolation_probability
+        # self.isolation_effectiveness = isolation_effectiveness
+        # self.isolation_house_effectiveness = isolation_house_effectiveness
```

Comment:
Should this code be commented out? Do we need it at all? In general its not good practice to leave code commented out in published code